docs/user_guide/plotting.md

```diff
@@ -1,11 +1,341 @@
+outputs.
+```shell
+law run cf.PlotVariables2D --version v1 \
+    --processes tt,st --variables n_jet-jet1_pt,jet1_pt-n_jet
+```
+
+::::{grid} 1 1 2 2
+:::{figure} ../plots/cf.PlotVariables2D_tpl_config_analy__1__b27b994979__plot__proc_2_a2211e799f__cat_incl__var_jet1_pt-n_jet.pdf
+:width: 100%
+:::
+
+:::{figure} ../plots/cf.PlotVariables2D_tpl_config_analy__1__b27b994979__plot__proc_2_a2211e799f__cat_incl__var_n_jet-jet1_pt.pdf
+:width: 100%
+:::
+::::
+
+For more information on the task parameters of the ```PlotVariables2D``` task, take a look into the
+[plotting task overview](../task_overview/plotting.md).
+
+## Creating cutflow plots
+
+The previously discussed plotting functions only create plots after applying the full event selection.
+To allow inspecting and optimizing an event and object selection, Columnflow also includes plotting
+tasks that can be run before applying any event selections.
+
+To create a simple cutflow plots, displaying event yields after each individual selection step,
+you can use the {py:class}`~columnflow.tasks.cutflow.PlotCutflow` task, e.g. via calling
+```shell
+law run cf.PlotCutflow --version v1 \
+    --calibrators example --selector example --categories incl,2j \
+    --shape-norm --process-settings tt,unstack:st,unstack \
+    --processes tt,st --selector-steps jet,muon
+```
+
+::::{grid} 1 1 2 2
+:::{figure} ../plots/cf.PlotCutflow_tpl_config_analy__1__12a17bf79c__cutflow__cat_incl.pdf
+:width: 100%
+:::
+
+:::{figure} ../plots/cf.PlotCutflow_tpl_config_analy__1__12a17bf79c__cutflow__cat_2j.pdf
+:width: 100%
+:::
+::::
+
+This will produce a plot with three bins, containing the event yield before applying any selection
+and after each selector step, where we always apply the locical and of all previous selector steps.
+
+To create plots of variables as part of the cutflow, we also provide the
+{py:class}`~columnflow.tasks.cutflow.PlotCutflowVariables1D`, which mostly behaves the same as the
+PlotVariables1D task.
+
+```shell
+law run cf.PlotCutflowVariables1D --version v1 \
+    --calibrators example --selector example \
+    --processes tt,st --variables cf_jet1_pt --categories incl \
+    --selector-steps jet,muon --per-plot processes
+```
+
+::::{grid} 1 1 3 3
+:::{figure} ../plots/cf.PlotCutflowVariables1D_tpl_config_analy__1__d8a37d3da9__plot__step0_Initial__proc_2_a2211e799f__cat_incl__var_cf_jet1_pt.pdf
+:width: 100%
+:::
+
+:::{figure} ../plots/cf.PlotCutflowVariables1D_tpl_config_analy__1__d8a37d3da9__plot__step1_jet__proc_2_a2211e799f__cat_incl__var_cf_jet1_pt.pdf
+:width: 100%
+:::
+
+:::{figure} ../plots/cf.PlotCutflowVariables1D_tpl_config_analy__1__d8a37d3da9__plot__step2_muon__proc_2_a2211e799f__cat_incl__var_cf_jet1_pt.pdf
+:width: 100%
+:::
+::::
+
+The ```per-plot``` parameter defines whether to produce one plot per selector step
+(```per-plot processes```) or one plot per process (```per-plot steps```).
```

Comment:
This seems very counter intuitive. Are you sure this isn't switched around?

Reply:
I agree that it's confusing, but that's how it is implemented. `per-plot processes` means that you get all processes per plot.

Reply:
OK then :D